RNIronSourceRewardedVideo.js

```diff
@@ -21,13 +21,20 @@ const addEventListener = (type, handler) => {
     case 'ironSourceRewardedVideoUnavailable':
     case 'ironSourceRewardedVideoDidOpen':
     case 'ironSourceRewardedVideoDidStart':
-    case 'ironSourceRewardedVideoClosedByUser':
     case 'ironSourceRewardedVideoClosedByError':
     case 'ironSourceRewardedVideoAdStarted':
     case 'ironSourceRewardedVideoAdEnded':
     case 'ironSourceRewardedVideoAdRewarded':
       eventHandlers[type].set(handler, IronSourceRewardedVideoEventEmitter.addListener(type, handler));
       break;
+    case 'ironSourceRewardedVideoClosedByUser':
+      eventHandlers[type].set(handler, IronSourceRewardedVideoEventEmitter.addListener(type, () => {
+        // This is a dirty hack that is required by some Ad Networks (Vungle, UnityAds)
+        // It makes 'ironSourceRewardedVideoClosedByUser' and 'ironSourceRewardedVideoAdRewarded'
+        // events order match with all other networks
+        setTimeout(handler, 1);
```

Comment:
is 1 mc enough? if you just need to send to the end of the callback queue, you can `setTimeout(handler)`